spec/base-spec.js

```diff
@@ -0,0 +1,43 @@
+/** @babel */
+
+/* This file contains all specs to ensure the base-functionality of
+this plugin. */
+
+import _path from 'path';
+// _helpers = require(_path.join(__dirname, '_spec-helpers.coffee'));
+
+const _projectRoot = _path.join(__dirname, 'fixtures');
+
+describe('editorconfig', () => {
+	beforeEach(() => {
+		waitsForPromise(() => {
+			return atom.packages.activatePackage('editorconfig');
+		});
+	});
+
+	describe('when providing base-settings', () => {
+		const filePath = _path.join(_projectRoot, 'base.txt');
+		var texteditor = null;
+
+		atom.workspace.open(filePath)
+    .then((newTexteditor) => {
+			texteditor = newTexteditor;
+		});
+
+		it('should have set the indent_style to "space"', () => {
+			expect(texteditor.getSoftTabs()).toBeTruthy();
+		});
+
+    it('should have set the indent_size to 4 characters', () => {
```

Comment:
Since editorconfig seems not working currently, Atom switched the intendation-chars. I fixed that.